public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -1,16 +1,16 @@
 import { isEqual } from 'lodash';
-import { finalize, from } from 'rxjs';
 
-import { Scope, ScopeNode } from '@grafana/data';
-import { config, getBackendSrv } from '@grafana/runtime';
-
-import { ScopesService } from '../ScopesService';
+import { ScopesApiClient } from '../ScopesApiClient';
 import { ScopesServiceBase } from '../ScopesServiceBase';
 import { ScopesDashboardsService } from '../dashboards/ScopesDashboardsService';
+import { getEmptyScopeObject } from '../utils';
 
 import { NodeReason, NodesMap, SelectedScope, TreeScope } from './types';
 
-interface ScopesSelectorServiceState {
+export interface ScopesSelectorServiceState {
+  loading: boolean;
```

Comment:
this loading is set only in this service when loading scopes so again seemed more appropriate here.